rdt-app/app/src/main/assets/json.form/patient-registration-form.json

```diff
@@ -0,0 +1,210 @@
+    "start": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "start",
+      "openmrs_entity_id": "163137AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "end": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "end",
+      "openmrs_entity_id": "163138AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "today": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "encounter",
+      "openmrs_entity_id": "encounter_date"
+    },
+    "deviceid": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "deviceid",
+      "openmrs_entity_id": "163149AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "subscriberid": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "subscriberid",
+      "openmrs_entity_id": "163150AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "simserial": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "simserial",
+      "openmrs_entity_id": "163151AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "phonenumber": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "phonenumber",
+      "openmrs_entity_id": "163152AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "encounter_location": ""
+  },
+  "step1": {
+    "title": "New client record",
+    "display_back_button": "true",
+    "next_label": "SAVE AND CONDUCT RDT",
+    "previous_label": "SAVE AND EXIT",
+    "bottom_navigation": "true",
+    "bottom_navigation_orientation": "vertical",
+    "previous": "",
+    "next": "",
+    "next_type": "submit",
+    "fields": [
+      {
+        "key": "patient_name_label",
+        "type": "label",
+        "text": "Name",
+        "text_color": "#000000"
+      },
+      {
+        "key": "patient_name",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "edit_type": "name",
+        "v_required": {
+          "value": "true",
+          "err": "Please specify patient name"
+        },
+        "v_regex": {
+          "value": "[^([0-9]*)$]*",
+          "err": "Please enter a valid name"
+        }
+      },
+      {
+        "key": "patient_age_label",
+        "type": "label",
+        "text": "Age",
+        "text_color": "#000000"
+      },
+      {
+        "key": "patient_age",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "edit_text",
+        "v_required": {
+          "value": "true",
+          "err": "Please specify patient age"
+        },
+        "v_numeric_integer": {
+          "value": "true",
+          "err": "Age must be a rounded number"
+        }
+      },
+      {
+        "key": "sex",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
```

Comment:
should be mapped to `person` and entity_id to `gender`

Reply:
Done.